tests/test_hierarchy_helpers.hpp

```diff
@@ -406,8 +406,10 @@ class TestMFMeshEvaluator final
                                            dealii::update_quadrature_points;
     std::shared_ptr<dealii::MatrixFree<dim, ScalarType>> mf_storage(
         new dealii::MatrixFree<dim, ScalarType>());
-    mf_storage->reinit(dof_handler, _agg_constraints,
-                       dealii::QGauss<1>(fe_degree + 1), additional_data);
+    mf_storage->reinit(
```

Comment:
Let's protect this code with a macro so that if deal.II is not patched we can still run the test. A simple macro defined in the file to switch the implementation is good enough.

Reply:
Good idea.